data/human/human missions.txt

```diff
@@ -4965,3 +4965,164 @@ mission "Riley"
+				"We have time, lets take the maglev."
+					goto maglev
+			label shuttle
+			`    After a short shuttle flight, you arrive in front of a single, not too large mountain, that peaks out of the rest of the landscape.`
+				goto front-mountain
+			label maglev
+			`    You take the maglev and get a tour through some regions of the planet. Although it is hard to get any detail because of the high speed, you get a good impression. After an hour you end up close to a mountain that stands solely in the landscape.`
+			label front-mountain
+			`    Liam, with help of the map, guides to a flat wall. He looks a bit embarrassed at you two, "Our father wants us to say: Edr- i fen. I do not know what it means, but I am most certain it refers to one of this fantasy novels he used to read."`
+			choice
+				"Sounds funny, lets do it."
+				"What a strange man, but I like his sense of humor."
+			`    You count to three and then shout out the phrase together. You all have to laugh a bit after that. Suddenly a blue line appears on the wall. The line shapes rectangle. After it has finished the shape, the stone inside the rectangle just disappears. All of you look surprised. "I could have sweared the stone was real," Liam says silently.`
+			choice
+				"Just like a real adventure, lets go!"
+					goto enter
+				"We do not know what is inside there, lets be careful."
+			`    "Oh, now that you say it, I recently bought a small camera drone," Liam says, "we can use it to scout out the hallway." He takes out a small box with a drone inside. "It is only in infrared, so we wont see much details," he adds. He starts the drone and guides it with a small remote control with integrated display. "I think it is safe, we can go," he tells you after some time.`
+			label enter
+			`    You enter the halway behind the opening. The walls are perfectly smooth and the light seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That is totally our dad." You proceed until you enter dome like room. On the walls all kind of alien artifacts line up next to each other. In the center of the room is a small slope with a strange looking sword inside. A small tablet stands on a socket in front of it.`
+			choice
+				"[get closer to the tablet]"
+					goto sword-scene
+				"[take a look around first]"
+			`    You walk around, looking at the various alien artifacts.`
+			`    A lot of the artifacts look similar to Sheragi technology, you ask yourself where he got this stuff.`
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`    After some time Sophie calls you both to the tablet.`
+			label sword-scene
+			scene "scene/sword dome"
+			`    You all get closer to the tablet. A short message is written on it:`
+			`    My dear children, this is the summary of all the things I have done, enjoy it!`
+			choice
+				"That is one of the best gifts I have ever seen."
+				"Now this is some cool stuff."
```

Comment:
```suggestion
```
I think best to take this part out. It will be a personal moment for the siblings.

Reply:
you are right